.github/actions/deploy/action.yml

```diff
@@ -106,14 +113,24 @@ runs:
         if-no-files-found: ignore
         retention-days: 7
 
+    - name: Evaluate job status
+      if: always()
+      shell: bash
+      run: |
+        if [[ ${{ join(steps.*.outcome, ',') }} =~ "failure" ]]; then
+          echo JOB_STATUS="failure" >> $GITHUB_ENV;
```

Comment:
I think semi colons are not necessary if there is a line break

Reply:
removed